### PR TITLE
make data safer by default.in case user delete pvc by accident , and limit RBAC hwameistor-role permissions

### PR DESCRIPTION
**Pull request by peng9808**
[make data safer by default ](https://github.com/hwameistor/hwameistor/pull/1519)
https://github.com/hwameistor/hwameistor/pull/1537

### CODE REVIEW

**Comment by peng9808** [comment]
@SSmallMonster Added helm set storageClassReclaimPolicy=Retain or  Delete...

File: helm/operator/templates/hwameistorcluster.yaml [hunk 1]
```diff
@@ -255,7 +255,7 @@ spec:
     chartVersion: "v0.4.1"
   storageClass:
     allowVolumeExpansion: true
-    reclaimPolicy: "Delete"
+    reclaimPolicy: {{ $.Values.storageClassReclaimPolicy }}
```
Comment (SSmallMonster) [inline helm/operator/templates/hwameistorcluster.yaml:258]: values.yaml should look like:
```yaml
storageClass:
  reclaimPolicy: "Retain"
```

**Review by SSmallMonster — CHANGES_REQUESTED** [review]

File: helm/operator/values.yaml [hunk 2]
```diff
@@ -183,6 +183,24 @@ dataLoadInit:
...
   tag: v0.0.1
 
+# StorageClass Settings
+storageClass:
+  # If enabled, a hwameistor storageclass will be created in your system
+  enable: true
+  # If enabled, hwameistor storageclass will be a default storageclass in system
+  # Attention: ensure your system doesn't have any default storageclass before
+  # more info: https://kubernetes.io/docs/tasks/administer-cluster/change-default-storage-class/
+  default: false
```
Comment (SSmallMonster) [inline helm/operator/values.yaml:193]: do operator use `enable` and `default` options now? if not, suggest remove it from values.yaml untill we support indeed.

**Review by SSmallMonster — APPROVED** [review]
lgtm

**Pull request merged**